acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#PTCA image
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:biweekly.202301.2
```

Comment:
Use template tags to keep this environment up to date: https://github.com/Azure/azureml-assets/wiki/Environments#template-tags

Reply:
why are we using AI infra as base image? Shouldn't we be using ACPT curated env?

Reply:
@Savita, you should not use CE image as a base image for another environment